arches/test/runner.py

```diff
@@ -0,0 +1,46 @@
+from django.test.runner import DiscoverRunner
+
+from arches.app.search.mappings import (
+    prepare_terms_index,
+    delete_terms_index,
+    prepare_concepts_index,
+    delete_concepts_index,
+    prepare_search_index,
+    delete_search_index,
+)
+from arches.app.utils.context_processors import app_settings
+
+
+class ArchesTestRunner(DiscoverRunner):
+    def __init__(self, *args, **kwargs) -> None:
+        kwargs["debug_mode"] = True
+        # Unless the user has something other than the Django default, give them
+        # what they probably want.
+        if kwargs["pattern"] == "test*.py":
+            kwargs["pattern"] = "*.py"
+        super().__init__(*args, **kwargs)
+
+    def setup_databases(self, **kwargs):
+        """Override DiscoverRunner's setup_databases() hook."""
+        ret = super().setup_databases(**kwargs)
+
+        # If the collected tests don't involve the database,
+        # e.g. subclasses of SimpleTestCase, then "aliases" will
+        # be None. Return early before attempting to setup
+        # something that doesn't exist.
+        if not kwargs.get("aliases"):
+            return ret
```

Comment:
I expanded this comment to make it more clear what was happening and added a docstring.